test/specs/reference/conf/reference.conf.json

```diff
@@ -1,5 +1,10 @@
 {
-    "catalog": {},
+    "catalog": {
+        "acls": [
```

Comment:
I'm not sure if its just not visible here but just in case... You'll need tests for various scenarios, and not just the anonymous case.